server/utils/articles/highlightvalidate.util.js

```diff
@@ -0,0 +1,33 @@
+import db from '../../models';
+import Response from '../response.util';
+
+const { Article } = db;
+
+class VerifyHighlightedText {
+/**
+ * @static
+ * @function VerifyHighlightedText
+ * @param {Object} req - request received
+ * @param {Object} res - response object
+ * @param {Object} next - response object
```

Comment:
Please I would like clarity on this `@param {Object} next - response object`.

Reply:
@samorahno Provide this clarity for the purpose of knowledge sharing

Reply:
@geekdanitek it is a callback function that calls the next middleware.